picard/ui/options/renaming.py

```diff
@@ -147,6 +155,32 @@ def _sync_scrollbar_vert(widget, value):
         # Sync example lists vertical scrolling
         sync_vertical_scrollbars((self.ui.example_filename_before, self.ui.example_filename_after))
 
+        # Set highlight colors for selected list items
+        example_style = self.ui.example_filename_before.palette()
+        highlight_bg = example_style.color(QPalette.Active, QPalette.Highlight)
+        highlight_fg = example_style.color(QPalette.Active, QPalette.HighlightedText)
+        stylesheet = "QListView::item:selected { color: " + highlight_fg.name() + "; background-color: " + highlight_bg.name() + "; }"
+        self.ui.example_filename_after.setStyleSheet(stylesheet)
+        self.ui.example_filename_before.setStyleSheet(stylesheet)
+
+        self.current_row = -1
+
+    def match_after_to_before(self):
+        if self.ui.example_filename_before.currentRow() != self.current_row:
+            self.current_row = self.ui.example_filename_before.currentRow()
+            self.ui.example_filename_after.setCurrentRow(self.current_row)
+
+    def match_before_to_after(self):
+        if self.ui.example_filename_after.currentRow() != self.current_row:
+            self.current_row = self.ui.example_filename_after.currentRow()
+            self.ui.example_filename_before.setCurrentRow(self.current_row)
+
```

Comment:
I think we can reduce code redundancy a bit here, also we need to block signals because `setCurrentRow()` will trigger change signal in turn (resulting in useless processing):

```python
    def match_after_before(self, source, target):
        """Matches selected item in target to source"""
        if source.currentRow() != self.current_row:
            self.current_row = source.currentRow()
            target.blockSignals(True)
            target.setCurrentRow(self.current_row)
            target.blockSignals(False)

    def match_after_to_before(self):
        """Sets the selected item in the 'after' list to the corresponding item in the 'before' list.
        """
        self.match_after_before(self.ui.example_filename_before, self.ui.example_filename_after)

    def match_before_to_after(self):
        """Sets the selected item in the 'before' list to the corresponding item in the 'after' list.
        """
        self.match_after_before(self.ui.example_filename_after, self.ui.example_filename_before)
```

@rdswift what do you think?

Reply:
I like it!  That piece of code always bothered me because it just didn't look right, but I wasn't seeing an easy way to fix it.  This does it.  Thanks.